### ISSUE TITLE
Missing some modules when built with Python 3

### ISSUE THREAD

**Issue by octacian** [wolfv/uberwriter#42]
When I attempt to build UberWriter with Python 3, as is suggested in the README, I receive errors regarding the absence of several Python modules. I installed all that I could through pip, however, some are still missing. I believe that this is because some of the modules are Python 2-specific (e.g. `StringIO`, apparently now contained within the `io` module). Not sure though, so I figured I'd open an issue here.

It is worth noting that UberWriter does launch properly, however, there is a several second delay between clicking the preview button and the preview actually rendering.

```
ERROR: Python module vim not found
ERROR: Python module ConfigParser not found
ERROR: Python module ConfigParser not found
ERROR: Python module StringIO not found
ERROR: Python module helpers not found
ERROR: Python module pylocales not found
```

Initially, there was a much longer list, but as mentioned I was able to install most of the missing modules through pip (e.g. `pressagio`). These modules, however, do not seem to exist in pip3.

**Comment by somas95** [comment]
Well, some things:
* Some of the packages mentioned are only imported if the installed python version is 2.0
I'll look why the error messages are triggered anyway. This application was initially developed before gtk3 and python3 if I recall correctly, so in the migrations some deprecated things may be left in the code
* Other packages (helpers, pylocales) are from Uberwriter itself, as I said I'll look in the error messages.
* Anyway, we intend to ship Uberwiter as flatpak, so all the dependencies will be selfcontained in the package. 
* Also, I think that these errors are caused by a bad setuptools configuration. I'm considering porting the build system to something like meson

* Regarding the delay switching to the preview panel; when clicking in that button a whole webview is instantiated, the html content created and then rendered. So, it's previsible that delay. It could be prevented having the webview always active and hidden, but that would increase the memory footprint of the application considerably.

**Comment by somas95** [comment]
Solved by #43
